### PR TITLE
cgo needs a compiler

### PR DESCRIPTION
**Pull request by jhheider**

### CODE REVIEW

File: projects/ipfs.tech/package.yml [hunk 1]
```diff
@@ -11,8 +11,16 @@ versions:
 build:
   dependencies:
     tea.xyz/gx/make: '*'
+    tea.xyz/gx/cc: c99
     go.dev: 1
+    gnu.org/patch: '*'
   script: |
+    if test "{{hw.platform}}" = "linux"; then
```
Comment (mxcl) [inline projects/ipfs.tech/package.yml:18]: This is just linux aarch64 right? We shouldn't apply fixes to more platforms than required.

Comment (mxcl) [reply projects/ipfs.tech/package.yml:18]: Also for all other go tools we do this in env, odd this requires a patch.

Comment (jhheider) [reply projects/ipfs.tech/package.yml:18]: yeah, it's because they don't pass any flags in their makefiles. I started with our SOP. ;)

Comment (jhheider) [reply projects/ipfs.tech/package.yml:18]: and our comments elsewhere say it's linux; I'll see if it's only aarch64.

Comment (jhheider) [reply projects/ipfs.tech/package.yml:18]: nope, x86_64 needs it too.

Comment (mxcl) [reply projects/ipfs.tech/package.yml:18]: oh ok.

**Pull request merged**